### PR TITLE
Ceph and Gluster cluster

### PR DESCRIPTION
**Pull request by dahorak**
Add playbook for preparing nodes for ceph and gluster cluster.
Add jobs for ceph and gluster cluster testing.

### CODE REVIEW

File: jobs/tendrl-cluster-test-jobs.yml [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+          # prepare configuration file
+          echo "" > usm.ini
+          echo "[usmqepytest]" >> usm.ini
+          echo "usm_log_level = DEBUG" >> usm.ini
+          echo "usm_username = admin" >> usm.ini
+          echo "usm_password = admin" >> usm.ini
+          echo "usm_web_url = https://${{usm_server}}" >> usm.ini
+          echo "usm_api_url = http://${{usm_server}}:9292/1.0/" >> usm.ini
+          echo "etcd_api_url = http://${{usm_server}}:2379/v2/" >> usm.ini
+          echo "usm_ca_cert = " >> usm.ini
```
Comment (mkudlej) [inline jobs/tendrl-cluster-test-jobs.yml]: This should be change by pul lrequest for usm.ini changes.

**Review by mkudlej — APPROVED** [review]
LGTM except of usm.ini generation.

File: jobs/tendrl-cluster-test-jobs.yml [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+          tests=""
+          for test in ${{TEST_LIST//[_,]/ }}; do
+            tests="${{tests}} usmqe_tests/api/tendrl/${{test}}";
+          done
+          # prepare env
+          ssh usmqe@${{qe_server}} "export PATH=\$PATH:\$HOME/.local/bin:\$HOME/bin; \
+            scl enable rh-python35 \
+              'pip install --user -r /home/usmqe/usmqe-tests/requirements.txt' ; \
+            source /opt/rh/rh-python35/enable ; \
+            export X_SCLS=\$(scl enable rh-python35 'echo $X_SCLS') ; \
```
Comment (mbukatov) [inline jobs/tendrl-cluster-test-jobs.yml:201]: I don't think enabling python35 softcol manually here is necessary, as `qe-server-user` role enables it in bashrc file of usmqe user.

**Review by mbukatov — COMMENTED** [review]
I don't have enough insight here, so I just added one suggestion related to software collections.

**Pull request merged**